app/models/cms/builtin.rb

```diff
@@ -117,6 +117,10 @@ def content_type
       'text/html'
     end
 
+    def liquid_enabled
+      true
+    end
```

Comment:
Just curious why is this needed.

Reply:
I forgot to answer to this one...

Honestly, I don't remember :sweat_smile:  let me check

Reply:
So, this was needed, because this test started failing:
https://github.com/3scale/porta/blob/rails70/test/unit/cms/builtin/page_test.rb#L33-L41

I figured out we also had this: https://github.com/3scale/porta/blob/bcbaea6392b58a89291932b2289127e44864e288/app/models/cms/builtin.rb#L57-L66
Which was the piece that made the test pass in Rails 6.

So, in Rails 6 the attribute was read like this:

![Screenshot from 2024-12-18 12-18-14](https://github.com/user-attachments/assets/f454a54f-0aca-4e73-a15d-08e9c1961351)
![Screenshot from 2024-12-18 12-18-23](https://github.com/user-attachments/assets/fcb634a3-8b31-4472-a9b8-27b61e0c5ea9)

And then it reached the overridden `read_attribute`.

In Rails 7 it's different. Now instead of `[]`, `public_send` is used:
![Screenshot from 2024-12-18 12-19-30](https://github.com/user-attachments/assets/5682e490-daae-49eb-a49d-6c2b485f3cfa)
And another difference is that instead of `read_attribute`, now `_read_attribute` is used. 
![Screenshot from 2024-12-18 12-19-39](https://github.com/user-attachments/assets/d0f03767-7a46-4a32-8265-702a051b032b)

So, I actually got rid of the old `read_attribute` overriden method because it's not used anymore, and moved this hardcoded method to `CMS::Builtin`.